docker-compose.yml

```diff
@@ -37,7 +37,7 @@ services:
       - '--web.console.templates=/usr/share/prometheus/consoles'
     ports:
       - 9090:9090
-    network_mode: "host"
+    #network_mode: "host"
     restart: always
 
   grafana:
```

Comment:
nit: We should pick a version number for this and pin it. Latest current major version okay with you?

Reply:
Yup sounds good.